src/Rules/NarrowPrivateClassMethodParamTypeRule.php

```diff
@@ -175,6 +175,7 @@ private function validateParam(Param $param, int $position, Expr $expr, Scope $s
         $errorMessage = sprintf(self::ERROR_MESSAGE, $position + 1, $argTypeAsString);
 
         return RuleErrorBuilder::message($errorMessage)
+            ->identifier('typePerfect.narrowPrivateClassMethodParamType')
```

Comment:
First tried naming like this: `type_perfect.no_empty_on_object`, but that results in `error identifiers must match /[a-zA-Z0-9](?:[a-zA-Z0-9\.]*[a-zA-Z0-9])?` so I ended up using camelCase.camelCase